modules/package.nix

```diff
@@ -215,6 +215,11 @@ in {
         type = bool;
         default = false;
       };
+
+      buildable = mkOption {
+        type = bool;
+        default = true;
```

Comment:
Is there a way we could instead set the default for the component to be the package setting? Then we wouldn't have to worry about having two things to check everywhere, which seems easy to get wrong.

Reply:
I tried that first, but could not get it to work.  I think because the default for the component is not actually used.  Instead component value is set by `plan-to-nix` to an expression based on the logic in the `.cabal` file.

Reply:
Okay, well at least we probably ought to factor out an `isBuildable comp` helper that we can use in various places so we don't miss something. Having two places to check just makes me nervous.